.github/workflows/full_stack_tests.yml

```diff
@@ -326,25 +282,12 @@ jobs:
         uses: actions/checkout@v4
       - name: Merge develop and set up Python
         uses: ./.github/actions/merge-develop-and-setup-python
-        with:
-          use_cache: true
-      # Caching is disabled to avoid poisoning our cache with the
-      # changed lighthouse file (see later).
-      #
-      # - name: Cache dependencies
-      #   uses: actions/cache@v4
-      #   with:
-      #     path: |
-      #       /home/runner/work/oppia/oppia_tools
-      #       /home/runner/work/oppia/yarn_cache
-      #       /home/runner/work/oppia/oppia/third_party
-      #     key: ${{ runner.os }}-oppiadeps-${{ hashFiles('yarn.lock', 'dependencies.json', 'requirements.txt', 'requirements_dev.txt') }}
-      #     restore-keys: |
-      #       ${{ runner.os }}-oppiadeps-
-      #       ${{ runner.os }}-
       - name: Install Oppia dependencies
         uses: ./.github/actions/install-oppia-dependencies
-      - name: Increase lighthouse timeout
+      - name: Install Chrome
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        uses: ./.github/actions/install-chrome
+      - name: Patch lighthouse to increase lighthouse timeout
         run: |
           # Replace the 1.5 second timeout in lighthouse for retrieving
```

Comment:
For future reference: this patch apparently has no effect at all. I output the lighthouse file after this step using `cat` and the timeout was still 1500.  So I'm dropping it completely.

As a note, if we need to do this patch in future, then it is important that we unpatch it in the same job step as well and log the output to Actions to verify that it has been handled correctly.

The job run is at https://github.com/oppia/oppia/actions/runs/12547787427/job/34985980152?pr=21536 and I've attached the corresponding job log (starting from the "patch lighthouse" step) for reference.
[job-logs.txt](https://github.com/user-attachments/files/18275564/job-logs.txt)